cmd/nf-dump/main.go

```diff
@@ -40,16 +40,21 @@ func main() {
 		log.Fatal(err)
 	}
 
+	if err = server.SetReadBuffer(262144); err != nil {
+	    log.Fatal(err)
+	}
+
 	decoders := make(map[string]*netflow.Decoder)
 	for {
 		buf := make([]byte, 8192)
 		var remote *net.UDPAddr
-		if _, remote, err = server.ReadFromUDP(buf); err != nil {
+		var cnt int
+		if cnt, remote, err = server.ReadFromUDP(buf); err != nil {
 			log.Printf("error reading from %s: %v\n", remote, err)
 			continue
 		}
 
-		log.Printf("received %d bytes from %s\n", len(buf), remote)
+		log.Printf("received %d bytes from %s\n", cnt, remote)
```

Comment:
In addition to reporting the correct number of bytes received, I think its also important to slice `buf` to `cnt` on line 66 to ensure the decoder only receives the actual bytes that came on the wire. Otherwise the decoder gets a bunch of zero bytes from `buf` that may or may not cause problems (v9 had an <a href="https://github.com/tehmaze/netflow/pull/6">issue with this</a>)